### PR TITLE
Update minimum-requirements.md

### PR DESCRIPTION
**Pull request by msbemba**
Added- Microsoft Business Premium and - Microsoft 365 E3 (M365 E3) with Addon Enterprise Mobility + Security E5

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6842

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/minimum-requirements.md [hunk 1]
```diff
@@ -44,7 +44,9 @@ Microsoft Defender for Endpoint requires one of the following Microsoft volume l
 - Windows 10 Enterprise E5
 - Windows 10 Education A5
 - Microsoft 365 E5 (M365 E5) which includes Windows 10 Enterprise E5 or Windows 11 Enterprise E5
-- Microsoft 365 A5 (M365 A5)
+- Microsoft 365 A5 (M365 A5) 
+- Microsoft 365 E3 (M365 E3) with Addon Enterprise Mobility + Security E5
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/minimum-requirements.md]:
```suggestion
- Microsoft 365 E3 (M365 E3) with Enterprise Mobility + Security E5 Add-on
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Pull request merged**